app/assets/stylesheets/common/base/menu-panel.scss

```diff
@@ -16,7 +16,7 @@
   // positions are relative to the .d-header .panel div
   top: 100%; // directly underneath .panel
   right: -10px; // 10px to the right of .panel - adjust as needed
-  max-height: 80vh;
+  max-height: calc(100vh - var(--header-offset) - 1em);
```

Comment:
A slightly unrelated change. `80vh` here was overriding the header calculations, and in some cases cutting off the menu quite early. 

Before/after

![image](https://github.com/user-attachments/assets/b15daf6f-8595-4123-8c65-f1cb8e116d60)

![image](https://github.com/user-attachments/assets/8fbada00-29f4-40d6-8560-28d2a93fd332)

Reply:
Should we use dvh here, to ensure the menu won't overflow on devices with dynamically showing/hiding browser chrome? (i.e. iOS)

Reply:
Sure, I don't see a downside to using `dvh`.